roles/dcos_requirements/tasks/main.yml

```diff
@@ -188,10 +188,13 @@
     policy: targeted
     state: "{{ dcos['selinux_mode'] | default('permissive') }}"
 
-- name: "Ensure groups 'nogroup' and 'docker' exist"
+- name: "Ensure 'docker' group exists"
   group:
-    name: "{{ item }}"
+    name: docker
     state: present
-  with_items:
-    - nogroup
-    - docker
+
+- name: "Ensure 'nogroup' group exists"
+  group:
+    name: nogroup
+    state: present
+  when: dcos_version_specifier is version_compare('2.0', '<')
```

Comment:
looking much better but IMO you should use dcos['version'] instead as this variable is not yet set.

Reply:
Fixed